hack/.ci/run-e2e-gke.sh

```diff
@@ -11,12 +11,20 @@ if [ -z "${ARTIFACTS+x}" ]; then
   exit 2
 fi
 
+if [ -z "${KUBECONFIG_DIR+x}" ]; then
+  kubeconfigs=("${KUBECONFIG}")
+else
+  kubeconfigs=()
+  for f in $( find "$( realpath "${KUBECONFIG_DIR}" )" -maxdepth 1 -type f -name '*.kubeconfig' ); do
+    kubeconfigs+=("${f}")
+  done
+fi
```

Comment:
I think this would be a good function in the library to get unified access in the other functions so they can always usu the kubeconfigs array even for single KUBECONFIG

Reply:
I moved it to a func but I'm not convinced about using it in other functions by default, you don't always want to call them for all clusters, so you loose some level of control.

Reply:
Ok, added this to two other functions. I'm still not convinced this is the right way to go though, it feels like it doesn't belong as part of a lib.